### PR TITLE
4296: added migrations for batched and recalled statuses

### PR DESCRIPTION
**Pull request by sutt0n**
migrations for #4540

### CODE REVIEW

File: shared/src/business/entities/cases/Case.js [hunk 1]
```diff
@@ -82,20 +82,36 @@ Case.STATUS_TYPES_MANUAL_UPDATE = [
...
+  other: 'Other',
+  partnershipSection1101: 'Partnership (BBA Section 1101)',
+  partnershipSection6226: 'Partnership (Section 6226)',
+  partnershipSection6228: 'Partnership (Section 6228)',
+  passport: 'Passport',
+  whistleblower: 'Whistleblower',
+  workerClassification: 'Worker Classification',
+};
+
 Case.CASE_TYPES = [
```
Comment (kkoskelin) [inline shared/src/business/entities/cases/Case.js]: Could consider here `Case.CASE_TYPES = Object.values(Case.CASE_TYPES_MAP);` I think

File: web-api/migrations/00012-case-status-in-progress.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const { Case } = require('../../shared/src/business/entities/cases/Case');
+const { isCaseRecord, upGenerator } = require('./utilities');
+
+const mutateRecord = item => {
+  if (
+    isCaseRecord(item) &&
+    ['Batched for IRS', 'Recalled'].includes(item.status)
+  ) {
+    item.status = Case.STATUS_TYPES.inProgress;
```
Comment (rachaelparris) [inline web-api/migrations/00012-case-status-in-progress.js]: Just to follow the pattern we've been using in other places and be a little more cautious, could you instantiate a new Case here before returning the item? You can look at `00005-service-indicator-respondents.js` for an example.

**Review by kkoskelin — APPROVED** [review]

File: web-api/migrations/00012-case-status-in-progress.js (hunk 2 above)
Comment (sutt0n) [reply web-api/migrations/00012-case-status-in-progress.js]: good catch, ty!

**Review by rachaelparris — APPROVED** [review]

**Review by rachaelparris — CHANGES_REQUESTED** [review]
Actually the way you're returning the entity won't work, because the pk won't be returned after it goes through the case constructor. You'll need to return 
``` 
{
...item,
...caseEntity
}
```

**Review by rachaelparris — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**